euth/dashboard/templatetags/form_tags.py

```diff
@@ -0,0 +1,36 @@
+from django import template
+from django.conf import settings
+
+from adhocracy4 import phases
+
+register = template.Library()
+
+
+@register.assignment_tag
```

Comment:
Assignment tags are deprecated since 1.9, simple tags can now store stuff in variables. Or do we need backwards compatibility here?